src/utils.ts

```diff
@@ -125,3 +165,7 @@ export function getFullImageName(image: string, tag: string): string {
     }
     return `${image}:${tag}`;
 }
+
+export function getTagSuffix(item: string): string {
```

Comment:
it is not clear what this function does without going to look at the call sites. It could have a better name like `removeIllegalCharacters` or `toValidImageName` or some combination of those